<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Base Calculator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="equationLine">
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="errorField">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(211, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item row="1" column="3">
       <widget class="QPushButton" name="pBtn_OpAdd">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(57, 57, 170);
color: white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pBtn_OpSub">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(57, 57, 170);
color: white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pBtn_7">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pBtn_leftBrace">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(122, 122, 122);
color:white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>(</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pBtn_6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="pBtn_1">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="pBtn_2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="pBtn_3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pBtn_FuncSqrt">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 109, 0);
color:white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>sqrt</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pBtn_5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QPushButton" name="pBtn_OpDiv">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(57, 57, 170);
color: white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pBtn_Backspace">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(247, 151, 34);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pBtn_8">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pBtn_FuncPow2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 109, 0);
color:white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>x^2</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QPushButton" name="pBtn_OpMult">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(57, 57, 170);
color: white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pBtn_9">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pBtn_4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="pBtn_Point">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>.</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QPushButton" name="pBtn_Calculate">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(166, 36, 19);
color: white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QPushButton" name="pBtn_Cancel">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(247, 151, 34);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>C</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pBtn_rightBrace">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(122, 122, 122);
color:white;
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QPushButton" name="pBtn_0">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(197, 197, 197);
font: 14pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
